scripts/generate_ipfs_hashes.py

```diff
@@ -40,12 +40,17 @@
 
 import yaml
 
-from aea.cli.common import _compute_fingerprint
+from aea.configurations.base import _compute_fingerprint
 from aea.helpers.ipfs.base import IPFSHashOnly
 
 AUTHOR = "fetchai"
 CORE_PATH = "aea"
-CORE_PACKAGES = {"connections": "stub", "protocols": "default", "skills": "error"}
+CORE_PACKAGES = {
+    "contracts": ["scaffold"],
```

Comment:
scaffold should not be fingerprinted imo. Or what's the rational?

Reply:
because `AEABuilder.add_component`, in the current implementation, always checks the consistency of the configuration when a package is added. If the scaffold packages are a package, then they should be consistent since the beginning.